packages/lwt-windows.2.5.2/opam

```diff
@@ -0,0 +1,61 @@
+opam-version: "1.2"
+maintainer: "Pierre Boutillier <pierre.boutillier@laposte.net>"
+authors: [
+  "Jérôme Vouillon"
+  "Jérémie Dimino"
+]
+homepage: "http://github.com/ocsigen/lwt"
+dev-repo: "git://github.com/ocsigen/lwt"
+bug-reports: "http://github.com/ocsigen/lwt/issues"
+license: "LGPL with OpenSSL linking exception"
+
+build: [
+  ["ocaml" "setup.ml" "-configure"
+    "--override" "os_type" "Win32"
+    "--override" "flexlink" "%{lib}%/flexdll-windows/flexlink"
+    "--override" "ext_dll" ".dll"
+    "--override" "datarootdir" "%{prefix}%/windows-sysroot/share"
+    "--disable-pthread"
+    "--prefix" "%{prefix}%/windows-sysroot"
+    "--disable-libev"
+#    "--%{camlp4:enable}%-camlp4"
+    "--disable-camlp4"
+    "--%{react-windows:enable}%-react"
+    "--%{ssl-windows:enable}%-ssl"
+    "--%{base-unix:enable}%-unix"
```

Comment:
This should just be always enabled, for the same reason we don't have a dependency or a depopt.